lib/index.js

```diff
@@ -215,7 +227,7 @@ Segment.prototype.normalize = function(msg) {
   }
   // if user provides campaign via context, do not overwrite with UTM qs param
   if (!ctx.campaign) {
-    ctx.campaign = query ? utm(query) : {};
+    ctx.campaign = query ? Object.assign(getVclid(query), utm(query)) : getVclid('');
```

Comment:
Why do we need `getVclid('')` here if it will always return an empty `{}` we should just `{}` like the original version instead of calling `getVclid('')`

Reply:
We cannot revert back to the original version because if the vclid is not found in the query, but only exists in session storage, we still need to send the vclid.

Reply:
Oh yup that makes sense!